client/src/styles/index.scss

```diff
@@ -1,5 +1,7 @@
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&display=swap');
-
+@import '~normalize.css';
+@import '~@blueprintjs/core/lib/css/blueprint.css';
+@import '~@blueprintjs/icons/lib/css/blueprint-icons.css';
```

Comment:
Now we are working with Material UI, I think that should be removed/replaced. Also, this may make main-sidebar links blue again

Reply:
Need to talk with couches